standard/docs/en/implementation/milestones.md

```diff
@@ -0,0 +1,52 @@
+# Milestones
+
+Milestones can be included within the planning, tender, contract and contract implementation blocks. 
+
+* The planning milestone block may describe anticipated milestones during the planning stage, such as the preparation 
+of key studies. Usually using the 'preProcurement' milestone type code.
+
+* Tender milestone blocks should be used to describe anticipated milestones during the proposed delivery 
+of a contract, as well as key dates from the process of tendering and awarding the contract.
+
+* Contract milestone block should be used for events related to the signing of the contract documents.
+
+* Contract implementation milestones block should be used for any event related to delivery of the contract.
+
```

Comment:
It might be clearer to have a consistent way of referring to milestone blocks in this list.

Either "the planning milestone block; the tender milestone block" .... etc.
OR "Planning milestone blocks; Tender milestone blocks; Contract milestone blocks" etc.

The former is probably more consistent with the Getting Started guidance on milestone as a re-usable block?